### PR TITLE
LeaderBoard-Hit-the-Api

### PR DESCRIPTION
**Pull request by kelomo2502**
#In this project : 

- I Implemented the form "Submit" button (sending data to the API).

- Implemented the "Refresh" button (receiving data from the API and parsing the JSON)
- Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/render.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  const container = document.querySelector('ul');
+  arr.forEach((game, index) => {
+    const li = document.createElement('li');
+    li.textContent = `${game.user}: ${game.score}`;
+    li.id = index;
+    container.appendChild(li);
+  });
+};
+
+export default render;
```
Comment (TedLivist) [inline src/render.js]: - [ ] There is an issue with your rendering. When I click the `Refresh` button, it appends the fetched scores to the existing ones, resulting in multiple scores. I would recommend you fix this.

  **SUGGESTION:** You can empty the container element by setting its innerHTML to an empty string before rendering the fetched scores, like so:
  ```
  const render = (arr) => {
    const container = document.querySelector('ul');
    container.innerHTML = ''
    ...
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @kelomo2502,

Good job so far!
✔️ Use of async and await
✔️ Scores can be added
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] In your README documentation, kindly include more specific instructions on how set up and run the app locally. After changing into the directory, users need to also run `npm install` to install packages and `npm start` to start the app

  kelono-readme

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/render.js (hunk 1 above)
Comment (kelomo2502) [reply src/render.js]: Thank you for the feedback. 
Will implement the required changes.
Thank you.

**Review by brytebee — APPROVED** [review]
Hi, @kelomo2502 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**